suite/runsuite.cmake

```diff
@@ -550,14 +552,17 @@ if (ARCH_IS_X86 AND UNIX AND (a64_on_x86_only OR NOT arg_automated_ci))
     " OFF ${arg_package} "")
 endif ()
 
-if (ARCH_IS_X86 AND UNIX AND cross_riscvxx_linux_only)
-# Do riscv cross-compilation if cross_riscvxx_linux_only is on.
+if (ARCH_IS_X86 AND UNIX)
+  #TODO i#3544: Run tests under QEMU
   set(orig_extra_ctest_args ${extra_ctest_args})
   set(run_tests OFF)
   set(prev_optional_cross_compile ${optional_cross_compile})
   set(prev_run_tests ${run_tests})
+  if (NOT cross_riscv64_linux_only)
+    set(optional_cross_compile ON)
+  endif ()
   set(ARCH_IS_X86 OFF)
-  # TODO Because of riscv can't run the build tests. So set it closed here.
+  #TODO i#3544: Run build tests.
```

Comment:
Same here. The space between the `#` and `TODO`.

Reply:
Done.